docs/excel/custom-functions-naming.md

```diff
@@ -47,17 +51,100 @@ To fix a naming conflict, change the `name` in your add-in and try the function
 
 [!include[manifest guidance](../includes/manifest-guidance.md)]
 
-## Localizing function names
-
-You can localize your function names for different languages using separate JSON files and override values in your add-in's manifest file. Avoid giving your functions an `id` or `name` that is a built-in Excel function in another language as this could conflict with localized functions.
-
-For full information on localizing, see [Localize custom functions](custom-functions-localize.md)
+## Localize custom functions
+
+You can localize both your add-in and your custom function names. To do so, provide localized function names in the functions' JSON file and locale information in the XML manifest file.
+
+>[!IMPORTANT]
+> Autogenerated metadata doesn't work for localization so you need to update the JSON file manually. To learn how to do this, see [Manually create JSON metadata for custom functions](custom-functions-json.md)
+
+### Localize function names
+
+To localize your custom functions, create a new JSON metadata file for each language. In each language JSON file, create `name` and `description` properties in the target language. The default file for English is named **functions.json**. Use the locale in the filename for each additional JSON file, such as **functions-de.json** to help identify them.
+
+The `name` and `description` appear in Excel and are localized. However, the `id` of each function isn't localized. The `id` property is how Excel identifies your function as unique and shouldn't be changed once it is set.
+
+>[!IMPORTANT]
+> Avoid giving your functions an `id` or `name` that is a built-in Excel function in another language as this could conflict with localized functions.
```

Comment:
Since this will conflict, let's remove the "could".
```suggestion
> Avoid giving your functions an `id` or `name` that is a built-in Excel function in another language as this conflicts with localized functions.
```